cmd/osbuild-image-tests/context-managers.go

```diff
@@ -90,7 +91,25 @@ func withBootedQemuImage(image string, ns netNS, f func() error) error {
 			return fmt.Errorf("cannot close temporary cloudinit file: %#v", err)
 		}
 
-		qemuCmd := ns.NamespacedCommand(
+		var qemuCmd *exec.Cmd
+		// This command does not use KVM as I was unable to make it work in Beaker,
+		// once we have machines that can use KVM, enable it to make it faster
+		qemuAarch64Cmd := ns.NamespacedCommand(
+			"qemu-system-aarch64",
+			"-cpu", "host",
+			"-M", "virt",
+			"-m", "2048",
```

Comment:
Is 2 GBs of memory really needed?

Reply:
I don't know, that's what @gicmo suggested :).